npm/linux-arm64/package.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Add `"description"` and `"author"` fields.

Reply:
Have added it.